src/Samples/Toolkit.SampleApp.UWP/App.xaml

```diff
@@ -2,7 +2,6 @@
     x:Class="Esri.ArcGISRuntime.Toolkit.SampleApp.App"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:Esri.ArcGISRuntime.Toolkit.SampleApp"
```

Comment:
The UWP implementation supports dark mode with theme-specific colors, so I wanted to be able to test that in the samples. That necessitated changes here and in several of the existing UWP samples.